server/models/SiteMetadataModel/index.js

```diff
@@ -1,15 +1,36 @@
 import { collections } from '../../utils/mongoCollections'
 
+const queryOptions = {
+  upsert: true,
+  returnDocument: 'after',
+}
 const SiteMetadataModel = {
   findOne: async (db, query) =>
     await db.collection(collections.metadata).findOne(query),
-  upsert: async (db, query, metadataAttributes) =>
-    await db
-      .collection(collections.metadata)
-      .findOneAndUpdate(query, metadataAttributes, {
-        upsert: true,
-        returnDocument: 'after',
-      }),
+  upsert: async (db, query, metadataAttributes, newParticipant = {}) => {
+    const isParticipantUpdated = Object.keys(newParticipant)
```

Comment:
Did you test this? As written this is going to return an empty array by default, which is truthy. Why default this to an empty object?

![Screenshot 2024-04-04 at 2 50 13 PM](https://github.com/AMP-SCZ/dpdash/assets/2905145/7cf091f4-71d0-4c24-98d8-5ff95aa1a228)

Reply:
[ee57de6](https://github.com/AMP-SCZ/dpdash/pull/723/commits/ee57de6b742667ca37eb6092536f8d9f2ece95d7)